RetroBar/Controls/ShowDesktopButton.xaml.cs

```diff
@@ -0,0 +1,84 @@
+﻿using ManagedShell.Common.Helpers;
+using ManagedShell.Interop;
+using RetroBar.Utilities;
+using System;
+using System.Diagnostics;
+using System.Windows;
+using System.Windows.Controls;
+
+namespace RetroBar.Controls
+{
+    /// <summary>
+    /// Interaction logic for ShowDesktopButton.xaml
+    /// </summary>
+    public partial class ShowDesktopButton : UserControl
+    {
+        private const int TOGGLE_DESKTOP = 407;
+        private IntPtr taskbarHandle = Process.GetCurrentProcess().MainWindowHandle;
+        private bool isWindows81OrBetter = EnvironmentHelper.IsWindows81OrBetter;
+
+        public ShowDesktopButton()
+        {
+            InitializeComponent();
+        }
+
+        private void ContextMenu_Opened(object sender, RoutedEventArgs e)
+        {
+            PeekAtDesktopItem.IsEnabled = true;
+            if (!NativeMethods.DwmIsCompositionEnabled())
+            {
+                PeekAtDesktopItem.IsEnabled = false;
+            }
+        }
+
+        private void ToggleDesktop()
+        {
+            NativeMethods.SendMessage(WindowHelper.FindWindowsTray(IntPtr.Zero),
```

Comment:
I think it should be better to add this to ManagedShell instead of being here.

Reply:
I agree. This could complement the other Aero Peek helper methods in ManagedShell.